cmd/lakefs/application/lakefscmdcontext.go

```diff
@@ -0,0 +1,21 @@
+package application
+
+import (
+	"context"
+	"github.com/treeverse/lakefs/pkg/config"
+	"github.com/treeverse/lakefs/pkg/logging"
+)
+
+type LakeFsCmdContext struct {
+	ctx    context.Context
```

Comment:
I believe [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments#contexts) suggest contexts should not be added as a struct member

Reply:
This is a great feedback, thanks. Did you use a tool to find out?

Reply:
No tool was used. This one came up in a conversation with @nopcoder, which suggested this standard